samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -148,28 +156,31 @@ public static TestRunner of(StreamApplication app) {
   }
 
   /**
-   * Only adds a config from {@code config} to samza job {@code configs} if they dont exist in it.
-   * @param config configs for the application
+   * Adds a config to Samza application, this config takes precedence over default configs and descriptor generated configs
```

Comment:
Use period instead of comma for unrelated sentences. "Adds a ... application. This config ..."